CHANGELOG.md

```diff
@@ -5,11 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.18.0] - 2023-08-29
+
+### Added
+
+- Implement V3.GET_PERMISSIONS
+
 ## [1.17.1] - 2023-05-08
 
 ### Fixed
 
-- readd V1.CLOSE Event, since it is still used internally
+- read V1.CLOSE Event, since it is still used internally
```

Comment:
I believe this is correct as 'readd'. It's supposed to be re-add, as in adding once again.